package.json

```diff
@@ -1,35 +1,66 @@
 {
-	"name": "acai",
-	"description": "The monorepo responsible for handling all of açaí's modules into a centralized space",
-	"author": "APO Softworks",
-	"license": "BSD-3-Clause",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/AcaiJS/modules"
-	},
-	"private": true,
-	"workspaces": {
-		"packages": [
-			"packages/*"
-		]
-	},
-	"devDependencies": {
-		"eslint": "^7.31.0",
-		"husky": "^7.0.0",
-		"syncpack": "^5.7.11"
-	},
-	"scripts": {
-		"aconfig": "cd ./packages/config && yarn",
-		"interfaces": "cd ./packages/interfaces && yarn",
-		"model": "cd ./packages/model && yarn",
-		"preset": "cd ./packages/preset && yarn",
-		"query": "cd ./packages/query && yarn",
-		"router": "cd ./packages/router && yarn",
-		"server": "cd ./packages/server && yarn",
-		"stubber": "cd ./packages/stubber && yarn",
-		"testing": "cd ./packages/testing && yarn",
-		"validator": "cd ./packages/validator && yarn",
-		"prepare": "husky install",
-		"lint": "yarn eslint"
-	}
-}
\ No newline at end of file
+  "name": "acai",
+  "description": "The monorepo responsible for handling all of açaí's modules into a centralized space",
+  "author": "APO Softworks",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/AcaiJS/modules"
+  },
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "devDependencies": {
+    "eslint": "^7.31.0",
+    "husky": "^7.0.0",
+    "lerna": "^4.0.0",
+    "syncpack": "^5.7.11"
+  },
+  "scripts": {
+    "all:publish": "yarn lerna publish from-git --yes",
+    "all:test": "yarn lerna run test --stream",
+    "all:version": "yarn lerna version",
+    "config:build": "yarn workspace @acai/config build",
+    "config:prepublishOnly": "yarn workspace @acai/config prepublishOnly",
+    "config:test": "yarn workspace @acai/config test",
+    "interfaces:build": "yarn workspace @acai/interfaces build",
+    "interfaces:lint:fix": "yarn workspace @acai/interfaces lint:fix",
+    "interfaces:lint": "yarn workspace @acai/interfaces lint",
+    "interfaces:prepublishOnly": "yarn workspace @acai/interfaces prepublishOnly",
+    "lerna": "./node_modules/.bin/lerna",
+    "lint": "yarn eslint",
+    "model:build": "yarn workspace @acai/model build",
```

Comment:
Also, I added model command so you could access it through "currying", you can `yarn model build`, it would call yarn workspace @acai/model internally and suffix it with build, just as the command is doing, that would save us a few lines of code inside the package json, since we are probably going to need it later on.